perses/data/host-guest/prepare_host_guest.py

```diff
@@ -0,0 +1,188 @@
+import mdtraj as md
+import numpy as np
+import simtk.openmm as openmm
+import openeye.oechem as oechem
+import openeye.oeiupac as oeiupac
+import tempfile
+import os
+import openeye.oeomega as oeomega
+
+#used SAMPL6 sampling challenge inputs
+#git revision 93f642ef9e2c478985cd14f3084d899ae3ee1c6d
+host_guest_path = "/Users/grinawap/SAMPL6/host_guest/SAMPLing/CB8-G3-0/PDB/complex.pdb"
```

Comment:
Can you eliminate the hard-coded paths so that this could be run in this directory, with the required input files provided (and their provenance documented in a `README.md` in this directory)? Otherwise, it won't be easy to tweak/fix these files if needed in future.